.storybook/stories/useVideoTexture.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { useState } from 'react'
+import * as THREE from 'three'
+
+import { Setup } from '../Setup'
+
+import { Plane, useVideoTexture, useTexture } from '../../src'
+
+export default {
+  title: 'Misc/useVideoTexture',
+  component: useVideoTexture,
+  decorators: [(storyFn) => <Setup>{storyFn()}</Setup>],
+}
+
+function VideoTexturedPlane() {
+  return (
+    <>
+      <Plane args={[4, 4]}>
+        <React.Suspense fallback={<FallbackMaterial url="images/sintel-cover.jpg" />}>
+          <VideoMaterial url="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/Sintel.mp4" />
+        </React.Suspense>
+      </Plane>
+    </>
+  )
+}
+
+function VideoMaterial({ url }) {
+  const texture = useVideoTexture(url, {})
+  return <meshBasicMaterial side={THREE.DoubleSide} map={texture} toneMapped={false} />
+}
+
+function FallbackMaterial({ url }: { url: string }) {
+  const texture = useTexture(url)
+  return <meshBasicMaterial map={texture} toneMapped={false} />
+}
+
+function UseVideoTextureScene() {
+  return (
+    <React.Suspense fallback={null}>
+      <VideoTexturedPlane />
+    </React.Suspense>
+  )
+}
+
+export const UseVideoTextureSceneSt = () => <UseVideoTextureScene />
+UseVideoTextureSceneSt.story = {
+  name: 'Default',
+}
+
+function VideoTexturedPlane2() {
+  const [mediaStream, setMediaStream] = useState(new MediaStream())
+  const videoTexture = useVideoTexture(mediaStream, {})
+
+  return (
+    <>
+      <Plane
+        args={[4, 4]}
+        onClick={async (e) => {
+          const mediaStream = await navigator.mediaDevices.getDisplayMedia({ video: true })
+          setMediaStream(mediaStream)
+        }}
+      >
+        <meshBasicMaterial map={videoTexture} toneMapped={false} side={THREE.DoubleSide} />
+      </Plane>
+    </>
+  )
+}
+
+function UseVideoTextureScene2() {
+  return (
+    <React.Suspense fallback={null}>
+      <VideoTexturedPlane2 />
+    </React.Suspense>
+  )
+}
+
+export const UseVideoTextureSceneSt2 = () => <UseVideoTextureScene2 />
+UseVideoTextureSceneSt2.story = {
+  name: 'MediaStream',
```

Comment:
@drcmda I've add a `MediaStream` story, but I'm currently enable to make it work: the Plane does not appear, as soon as I `const videoTexture = useVideoTexture(mediaStream, {})` https://github.com/pmndrs/drei/pull/1370/files#diff-8dffe909d538f296844bc40940c1d1aef597cadfa7581d485421b73e61eed225R52